bot/modules/characters/commands/characters.js

```diff
@@ -0,0 +1,49 @@
+import { Command } from 'discord-utils';
+import { Characters } from '../../../../api/models'; // TODO: Add link-module-alias
+
+module.exports = class extends Command
+{
+  constructor()
+  {
+    super();
+    this.keyword = 'character';
+    this.aliases.push('c');
+  }
+
+  /** @param {import('discord-utils').Context} context */
+  async action(context)
+  {
+    const player = context.message.author.id;
+
+    /* Check if a character name or code was given. */
+    if(!context.parameters)
+      return context.send("❌  Please indicate the character's name or code.");
+    
+    /* The given character name or code. */
+    const characterQuery = context.parameters.shift();
+    this.charactersFound = await Characters.find(player, characterQuery);
```

Comment:
Is there really a need to declare this as a class attribute? I think declaring it as a block scoped variable would do just fine.

Reply:
Confirmed solution in https://github.com/TheRayOfSeasons/Beholder/pull/11/commits/e5a3bef2ecb20cba4cc07296db650ff1974ba398. Resolving this.